Module/rental_options.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../Components/rental'
+require_relative 'input'
+require_relative '../utils/io_file'
+
+class RentalOptions
+  attr_accessor :rentals_list
+
+  def initialize(book_options, people_options)
+    @book_options = book_options
+    @people_options = people_options
+    @rental_file = IOFile.new('./my-data/rentals.json')
+    @rentals_list = @rental_file.read_data
+  end
+
+  def select(label, option)
+    puts label
+    case option
+    when 'book'
+      @book_options.list_all_books
+    when 'person'
+      @people_options.list_all_people
+    end
+    gets.chomp.to_i
+  end
+
+  def create_rental
+    book_num = select('Select a book from the following list by number', 'book')
+    person_num = select('Select a book from the following list by number', 'person')
+
+    date = Input.user_input('Date: ')
+    @rentals_list.push(Rental.new(date, @book_options.books_list[book_num], @people_options.people_list[person_num]))
+    @rental_file.write_data(@rentals_list)
+    puts 'Rental created successfully'
+  end
+
+  def list_rentals
+    id = select('Select a person from the following list by id ', 'person')
+
+    puts 'Rentals:'
+    @rentals_list.each do |rental|
+      puts "Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}" if rental.person.id == id
+    end
```

Comment:
- [ ] @team great work on your RentalOptions class methods, however when I save a rental am unable to view it, kindly check on your output for the rentals 
![image](https://user-images.githubusercontent.com/61408860/234690578-cecc7333-cf14-471c-8bd5-89fad6be12a9.png)